openwisp_utils/admin_theme/apps.py

```diff
@@ -27,3 +30,15 @@ def register_menu_groups(self):
             position=10,
             config={'label': _('Home'), 'url': '/admin', 'icon': 'ow-dashboard-icon'},
         )
+
+    def modify_admin_theme_settings_links(self):
+        OPENWISP_ADMIN_THEME_LINKS = getattr(settings, 'OPENWISP_ADMIN_THEME_LINKS', [])
+        OPENWISP_ADMIN_THEME_JS = getattr(settings, 'OPENWISP_ADMIN_THEME_JS', [])
+
+        for css_files in OPENWISP_ADMIN_THEME_LINKS:
+            css_files['href'] = static(css_files['href'])
+
+        OPENWISP_ADMIN_THEME_JS = list(map(static, OPENWISP_ADMIN_THEME_JS))
+
+        setattr(settings, 'OPENWISP_ADMIN_THEME_LINKS', OPENWISP_ADMIN_THEME_LINKS)
+        setattr(settings, 'OPENWISP_ADMIN_THEME_JS', OPENWISP_ADMIN_THEME_JS)
```

Comment:
it feels weird to do this here and overwrite the django setting.

What about doing this preprocessing here instead (by moving these settings as last and create a function which does the preprocessing)?
https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/admin_theme/settings.py#L9-L10

Reply:
I tried doing so but it will not work as apps are not loaded in beginning and `static` function will not work.